run.py

```diff
@@ -98,6 +104,15 @@ def run_one_step(func, nwarmup=WARMUP_ROUNDS, model_flops=None, num_iter=10, mod
             func()
             t1 = time.time_ns()
             result_summary.append([(t1 - t0) / 1_000_000])
+        if stress != 0:
+            cur_time = time.time_ns()
+            # print out the status every 10s.
+            if (cur_time - last_time) >= 10 * 1e9:
+                est = (target_time - cur_time) / 1e9
+                print('{:<20} {:>20}'.format("Passed Iterations:", "%d" % _i))
```

Comment:
Let's make the two prints on the same line, and organize it like 
```suggestion
                | iter 20 | time per iter | rest time |
```

Reply:
fixed by https://github.com/pytorch/benchmark/pull/1002/commits/7ebb628fa0baa5333c53df6df5eca7d7b3bf11ac